tests/e2e/tools/lib/covers.js

```diff
@@ -12,8 +11,8 @@ async function addCoversBlock(page, editor, style = '') {
   // Select the style if needed.
   if (style) {
     const stylePicker = await page.locator('.block-editor-block-styles__variants');
-    // CSS selector needs single word,hence remove word after space(eg Take Action=>Take).
-    const label = style === 'Content' ? /Content|Default/ : style.split(' ')[0];
+    //CSS selector needs single word,hence remove word after space(eg Take Action=>Take).
+    const label = style.split(' ')[0];
```

Comment:
We should be able to always use `Take Action` as label now, instead of doing this